pyrs/calibration/peakfit_calibration.py

```diff
@@ -489,13 +502,13 @@ def FullCalibration(self, initalGuess=None):
         if initalGuess is None:
             initalGuess = self.get_calib()
 
-        out = least_squares(self.peaks_alignment_all, initalGuess, jac='3-point',
-                            bounds=([-.05, -.05, -.05, -np.pi/20, -np.pi/20, -np.pi/20, 1.4],
-                                    [.05, .05, .05, np.pi/20, np.pi/20, np.pi/20, 1.5]),
-                            method='dogbox', ftol=1e-08, xtol=1e-08, gtol=1e-08, x_scale=1.0,
-                            loss='linear', f_scale=1.0, diff_step=None, tr_solver='exact', tr_options={},
-                            jac_sparsity=None, max_nfev=None, verbose=0,
-                            args=(None, False), kwargs={})
+        out = self.FitDetector(self.peaks_alignment_all, initalGuess, jac='3-point',
```

Comment:
This is a really long list of parameters to remember to pass to a function. Using names was a good way to make it less confusing.